sql/hive/compatibility/src/test/scala/org/apache/spark/sql/hive/execution/HiveCompatibilitySuite.scala

```diff
@@ -221,9 +221,6 @@ class HiveCompatibilitySuite extends HiveQueryFileTest with BeforeAndAfter {
     "udf_when",
     "udf_case",
 
-    // Needs constant object inspectors
-    "udf_round",
```

Comment:
Not really, since #6835 is still open and not merged into mainstream. but the problem is NaN support rather than `Needs constant object inspectors ` which is no longer the issue.